algebird-test/src/test/scala/com/twitter/algebird/BloomFilterTest.scala

```diff
@@ -255,15 +255,13 @@ class BloomFilterTest extends WordSpec with Matchers {
         stream.toByteArray
       }
 
-      val items = (1 until 10).map { _.toString }
+      val items = (1 until 10).map(_.toString)
       val bf = BloomFilter(10, 0.1).create(items: _*)
-      val bytesBeforeSizeCalled = new String(serialize(bf))
+      val bytesBeforeSizeCalled = Bytes(serialize(bf))
       bf.size
-
       assert(bf.contains("1").isTrue)
-
-      val bytesAfterSizeCalled = new String(serialize(bf))
-      assert(bytesBeforeSizeCalled == bytesAfterSizeCalled)
+      val bytesAfterSizeCalled = Bytes(serialize(bf))
+      assert(bytesBeforeSizeCalled.size == bytesAfterSizeCalled.size)
```

Comment:
this is worrisome in that the `transient lazy val` trick is VERY commonly used to sidestep scalding serialization issues. We should probably chase it a bit or we will see massive pains on trying to merge this at scale.